src/rpc/blockchain.cpp

```diff
@@ -1743,7 +1747,7 @@ static UniValue getblockstats(const JSONRPCRequest& request)
                 {RPCResult::Type::NUM, "totalfee", "The fee total"},
                 {RPCResult::Type::NUM, "txs", "The number of transactions (including coinbase)"},
                 {RPCResult::Type::NUM, "utxo_increase", "The increase/decrease in the number of unspent outputs"},
-                {RPCResult::Type::NUM, "utxo_size_inc", "The increase/decrease in size for the utxo index (not discounting op_return and similar)"},
+                {RPCResult::Type::NUM, "utxo_size_inc", "The increase/decrease in size for the utxo index"},
```

Comment:
With this change to the documented API, I think you'd better add a new field instead...

Reply:
Slightly would have preferred to just fix the existing values but I have now introduced new values with a *_actual naming scheme.